runtime/packages/core-modules/assembly/log.ts

```diff
@@ -1,15 +1,43 @@
-import { Serialiser } from '@as-substrate/core-utils';
-import { ScaleString } from 'as-scale-codec';
-import { ext_misc_print_utf8_version_1 } from './env';
+import { Serialiser, Utils } from '@as-substrate/core-utils';
+import { ext_misc_print_utf8_version_1, ext_logging_log_version_1 } from './env';
 
 export class Log {
+    public static readonly LOG_TARGET: string = "runtime";
+    public static readonly ERROR_LEVEL: i32 = 0;
+    public static readonly WARN_LEVEL: i32 = 1;
     /**
-     * Prints a message to the Host
+     * Base function for logging
+     * @param logLevel 
+     * @param message 
+     */
+    static baseLog(logLevel: i32, message: string): void {
```

Comment:
You can indicate that this is a "private" function by renaming the function to `_baseLog`